src/main/java/blackjack/domain/card/CardFactory.java

```diff
@@ -0,0 +1,34 @@
+package blackjack.domain.card;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+public class CardFactory {
+    private final List<Card> cards;
+
+    private CardFactory() {
+        this.cards = Collections.unmodifiableList(generateAllCards());
+    }
+
+    private LinkedList<Card> generateAllCards() {
+        LinkedList<Card> cards = new LinkedList<>();
```

Comment:
```suggestion
        List<Card> cards = new LinkedList<>();
```
요구사항이 변경되서 LinkedList가 아닌 ArrayList를 써야한다면 수정해야될 부분이 많아지지 않을까요?